plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/TrinoMetricsReporter.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.iceberg;
+
+import io.airlift.log.Logger;
+import org.apache.iceberg.metrics.LoggingMetricsReporter;
+import org.apache.iceberg.metrics.MetricsReport;
+import org.apache.iceberg.metrics.MetricsReporter;
+
+/**
+ * Trino default {@link MetricsReporter} implementation.
+ * <p>
+ * Similar to Iceberg's default {@link MetricsReporter} implementation,
+ * the {@link LoggingMetricsReporter}, but does not log at {@code INFO} level,
+ * just eliminating log file size impact.
+ *
+ * @see LoggingMetricsReporter
+ */
+public enum TrinoMetricsReporter
+        implements MetricsReporter
+{
+    TRINO_METRICS_REPORTER;
+
+    private static final Logger log = Logger.get(TrinoMetricsReporter.class);
+
+    @Override
+    public void report(MetricsReport report)
+    {
+        log.debug("Received metrics report: %s", report);
```

Comment:
Maybe put "Iceberg" in the log prefix somewhere

Reply:
it's part of the log context (FQCN)